docs/distributed/README.md

```diff
@@ -1,74 +1,85 @@
+* All nodes running distributed Minio must be in a homogeneous environment (meaning they have the same operating system, number of drives, and interconnects).
+* If the domain must be set, the `MINIO_DOMAIN` environment variable must be defined and exported.
+* Minio distributed mode requires directories that are exclusive to Minio. For example, if a volume is mounted under `/export`, pass `/export/data` as an argument to the `minio server` command.
+**Note:** The drives can be shared with other applications if required.
+* The IP addresses and drive paths below are intended for demonstration purposes only, and must be replaced with the actual IP addresses and drive paths/folders.
+* The time on the servers running distributed Minio instances should within three seconds of each other. Use [NTP](http://www.ntp.org/) to ensure consistent times across servers.
+* Running Distributed Minio on Windows is experimental as of now. Please proceed with caution.
+
+#### Example
+
+This example demonstrates how to start distributed Minio instances on eight nodes, with one disk each mounted at `/export1`: 
 
-Example 1: Start distributed Minio instance on 8 nodes with 1 disk each mounted at `/export1` (pictured below), by running this command on all the 8 nodes:
 ![Distributed Minio, 8 nodes with 1 disk each](https://github.com/minio/minio/blob/master/docs/screenshots/Architecture-diagram_distributed_8.jpg?raw=true)
-#### GNU/Linux and macOS
+
+Run the following command on all eight nodes:
 
 ```sh
 export MINIO_ACCESS_KEY=<ACCESS_KEY>
 export MINIO_SECRET_KEY=<SECRET_KEY>
 minio server http://192.168.1.1{1...8}/export1
 ```
 
+**Note:** Arguments such as `{1...n}` have three dots. Using only two dots (e.g. `{1..4}`) will be interpreted by the shell and won't be passed to Minio Server. This affects the erasure coding order, possibly impacting performance and high availability. Always use `{1...n}` (three dots) to use Minio Server erasure code on data optimally.
 
-__NOTE:__ `{1...n}` shown have 3 dots! Using only 2 dots `{1..4}` will be interpreted by your shell and won't be passed to minio server, affecting the erasure coding order, which may impact performance and high availability. __Always use `{1...n}` (3 dots!) to allow minio server to optimally erasure-code data__
-
-## 3. Test your setup
-To test this setup, access the Minio server via browser or [`mc`](https://docs.minio.io/docs/minio-client-quickstart-guide).
+### <a name="test-the-configuration"></a>3. Test the Configuration
+To test this configuration, access the Minio server via a browser or invoke it as described in the [Minio Client Quickstart Guide](https://docs.minio.io/docs/minio-client-quickstart-guide).
```

Comment:
Do we need to mention how to check the browser?  Since we don't give an example of server startup output new users might not know how